src/main/java/com/sfuentes/Worker.java

```diff
@@ -15,8 +15,7 @@ public double calculateSalary() {
     if (hours <= 40)
       return hours * 200;
     else {
-      hours -= 40;
-      return 8_000 + hours * 250;
+      return 8_000 + (hours - 40) * 250;
```

Comment:
curly braces are not required in else since you removed the other line.

Reply:
fixed